opal/mca/btl/tcp/btl_tcp_proc.c

```diff
@@ -223,6 +221,10 @@ mca_btl_tcp_proc_t* mca_btl_tcp_proc_create(opal_proc_t* proc)
         }
     }
 
+    if (remote_addrs){
```

Comment:
I missed this in the internal review, but we explicitly check against NULL in Open MPI.  So this should be if (NULL == remote_addrs) {.

Reply:
Ah yes, I will change this.

Reply:
Oops, should be if (NULL != remote_addrs)